DEVELOPMENT.md

```diff
@@ -115,3 +139,49 @@ bazel build third_party/chromium
 
 # Lastly, put the address returnd by the web server into the Chromium.
 ```
+
+#### Running webapp tests
+
+tl;dr. Use `ibazel test` for regular work (supports limiting the test amount with
+`fit/fdescript` and `console.log` usage) and `ibazel run` for karma console breakpoint
+debugging.
+
+We use karma testing chromium target for our UI unit tests in webapp. However,
+depending on the environments (OSS or internal TB.corp), the commands we use and
+the functionalities are slightly different.
+
+1.  Just run all webapp tests. The job stops after finished. `console.log` is not
+    supported. Not handy on development.
+
+    ```shell
+    (tf)$ bazel test //tensorboard/webapp:karma_test_chromium-local
+    ```
+
+2.  Using `ibazel` to auto detect the file changes and use target
+    `karma_test_chromium-local` for running on *all* tests. Practically
+    `console.log` and `fit/fdescript` (which areused to narrow down the test amount) are
```

Comment:
I think fit/fdescribe work in all scenarios you describe in this section so not sure why its worth calling out here. Maybe move it to its own paragraph elsewhere?